lib/spans/streaming-span-event.js

```diff
@@ -273,12 +276,13 @@ class StreamingDatastoreSpanEvent extends StreamingSpanEvent {
       this.addAgentAttribute('db.instance', database_name)
     }
 
-    if (host) {
-      this.addAgentAttribute('peer.hostname', host)
-      this.addAgentAttribute('server.address', host)
+    // `host` was translated in StreamingSpanEvent class
+    // use `server.address` to assign to `peer.hostname`
+    if (agentAttributes.host) {
```

Comment:
Comment doesn't seem to match?

Reply:
ah, i had to change this so i never updated the comment. but like the feedback above this may not be needed